app/models/custom-form.js

```diff
@@ -4,44 +4,44 @@ import ModelBase from 'open-event-frontend/models/base';
 import { belongsTo, hasMany } from 'ember-data/relationships';
 
 export const SPEAKER_FORM_FIELDS = {
-  name                : 'Name',
-  email               : 'Email',
-  position            : 'Position',
-  organisation        : 'Organisation',
-  address             : 'Address',
-  city                : 'City',
-  country             : 'Country',
-  mobile              : 'Mobile',
-  gender              : 'Gender',
-  shortBiography      : 'Short Biography',
-  longBiography       : 'Long Biography',
-  speakingExperience  : 'Speaking Experience',
-  photoUrl            : 'Photo',
-  website             : 'Website',
+  name                : this.l10n.t('Name'),
+  email               : this.l10n.t('Email'),
+  position            : this.l10n.t('Position'),
+  organisation        : this.l10n.t('Organisation'),
+  address             : this.l10n.t('Address'),
+  city                : this.l10n.t('City'),
+  country             : this.l10n.t('Country'),
+  mobile              : this.l10n.t('Mobile'),
+  gender              : this.l10n.t('Gender'),
+  shortBiography      : this.l10n.t('Short Biography'),
+  longBiography       : this.l10n.t('Long Biography'),
+  speakingExperience  : this.l10n.t('Speaking Experience'),
+  photoUrl            : this.l10n.t('Photo'),
+  website             : this.l10n.t('Website'),
```

Comment:
Wrong

Reply:
Sir please can you provide me a clue for it,